std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
```

Comment:
ddoc doesn't redo indenting of code samples, just the left margin.

I redid all the examples in the language spec to be tighter and work on the Kindle, but they've been bit rotting since and things have been reverting to being unreadable on the Kindle.

Source code in Phobos we can presume is being read on a larger screen. But people read documentation on ereaders, meaning much shorter lines. Articles as well should use shorter lines. Nothing is uglier than line-wrapped code text.